http/java/okhttp/lib/src/main/java/com/microsoft/kiota/http/KiotaClientFactory.java

```diff
@@ -37,6 +41,11 @@ private KiotaClientFactory() { }
      */
     @Nonnull
     public static Interceptor[] CreateDefaultInterceptors() {
-        return new Interceptor[] {}; //TODO add the list of default interceptors when they are ready
+        return new Interceptor[] {
+            new RedirectHandler(),
+            new RetryHandler(),
+            new TelemetryHandler()
```

Comment:
```suggestion
```
This guy should probably be part of the graph core defaults, not the kiota defaults. Because not all kiota sdks are going to be calling graph.